content/millennium/dstu2/scheduling/appointment.md

```diff
@@ -55,3 +55,86 @@ List an individual Appointment by its id:
 
 <%= headers status: 200 %>
 <%= json(:dstu2_appointment_read) %>
+
+### Errors
+
+The common [errors] may be returned.
+
+
+## Create
+
+Create a new Appointment.
+
+    POST /Appointment
+
+_Implementation Notes_
+
+* [Appointment.status] must be set to `proposed`.
+* [Appointment.slot] must be a reference to the Slot in which this appointment is being booked.
+* [Appointment.participant] should have exactly one participant.
+* [Appointment.participant.status] must be set to `needs-action`.
+* [Appointment.participant.type] must not be set.
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, patient: true, system: true) %>
+
+### Headers
+
+<%= headers head: {Authorization: '&lt;OAuth2 Bearer Token>', Accept: 'application/json+fhir', 'Content-Type': 'application/json+fhir'} %>
+
+### Body Fields
+
+<%= definition_table(:appointment, :create, :dstu2) %>
+
+### Example
+
+#### Request
+
+    POST https://fhir-ehr.sandboxcerner.com/dstu2/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/Appointment/
+
+#### Body
+
+<%= json(:dstu2_appointment_create) %>
+
+#### Response
+
+<%= headers status: 201 %>
+<pre class="terminal">
+    Connection → Keep-Alive
+    Content-Encoding → gzip
+    Content-Length → 20
+    Content-Type → text/html; charset=UTF-8
+    Date → Wed, 13 Jan 2016 21:45:47 GMT
+    Keep-Alive → timeout=15, max=100
+    Last-Modified → Tue, 15 Dec 2015 19:13:20 GMT
+    Status → 201 Created
+    access-control-allow-methods → DELETE, GET, POST, PUT, OPTIONS, HEAD
+    access-control-allow-origin → *
+    access-control-expose-headers → ETag, Content-Location, Location, X-Request-Id, WWW-Authenticate, Date
+    access-control-max-age → 0
+    cache-control → no-cache
+    etag → W/"0"
+    location → https://fhir-ehr.sandboxcerner.com/dstu2/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/Appointment/20465903
+    server-response-time → 1260.984596
+    strict-transport-security → max-age=631152000
+    vary → Origin,User-Agent,Accept-Encoding
+    x-content-type-options → nosniff
+    x-frame-options → SAMEORIGIN
+    x-request-id → 682c633c-b20f-4f6f-8fae-c58b3aeffe04
+    x-runtime → 1.260940
+    x-xss-protection → 1; mode=block
+</pre>
+
+The `ETag` response header indicates the current `If-Match` version to use on subsequent updates.
+
+### Errors
+
+The common [errors] may be returned.
```

Comment:
Shouldn't we be returning error + operation outcomes for modifier extensions and implicit rules (at least)? That should also be called out in the overview.

Reply:
Addressed in https://github.com/cerner/fhir.cerner.com/pull/153/commits/3b3031eb9a5758fc6e3744a2db42762c9eaff938